src/compiler.js

```diff
@@ -104,6 +104,21 @@ class Compiler {
         })
       })
 
+      // Add mutation files
+      if (subgraph.has('mutations')) {
+        let mutations = subgraph.get('mutations')
+        files.push(mutations.getIn(['schema', 'file']))
+
+        let resolversKind = mutations.getIn(['resolvers', 'kind'])
+        switch (resolversKind) {
+          case "javascript":
+            files.push(mutations.getIn(['resolvers', 'file']))
+            break;
+          default:
+            throw Error(`Unimplemented resolvers kind '${resolversKind}'`)
```

Comment:
I think it's safe to assume that all resolver types would be defined in an external file. So we can drop the `switch` here.

We should catch unsupported resolver types when we validate the manifest.